benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -123,14 +134,39 @@ def _run_vm(self):
         self._set_bootstorm_vm_first_run_time()
         self._set_bootstorm_vm_start_time(vm_name=self._vm_name)
         self._virtctl.start_vm_sync(vm_name=self._vm_name)
-        self._data_dict = self._get_bootstorm_vm_elapsed_time(vm_name=self._vm_name)
+        self.vm_node = self._ssh_vm(vm_name=self._vm_name)
+        self._data_dict = self._get_bootstorm_vm_elapsed_time(vm_name=self._vm_name, vm_node=self.vm_node)
         self._data_dict['run_artifacts_url'] = os.path.join(self._run_artifacts_url,
                                                             f'{self._get_run_artifacts_hierarchy(workload_name=self._workload_name, is_file=True)}-{self._time_stamp_format}.tar.gz')
         self._finalize_vm()
         self._oc.delete_vm_sync(
             yaml=os.path.join(f'{self._run_artifacts_path}', f'{self._name}.yaml'),
             vm_name=self._vm_name)
 
+    def _verify_vm_ssh(self):
+        """
+        This method verifies each VM ssh login
+        :return:
+        """
+        try:
+            vm_names = self._oc._get_all_vm_names()
```

Comment:
It might make sense to compare this against a list of expected VM names, if that's practical here.  And you certainly want to make sure that there is at least one, or you'll simply be "verifying" a null case.

Reply:
You right, add MissingVMs error when no running VMs at all